engine/src/main/java/com/datatorrent/stram/client/StramClientUtils.java

```diff
@@ -118,6 +118,8 @@
   public static final long DELEGATION_TOKEN_MAX_LIFETIME_DEFAULT = 7 * 24 * 60 * 60 * 1000;
   public static final String TOKEN_REFRESH_PRINCIPAL = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.principal";
   public static final String TOKEN_REFRESH_KEYTAB = StramUserLogin.DT_AUTH_PREFIX + "token.refresh.keytab";
+  public static final String CUSTOM_SSL_KEYSTORE_CONFIG = StreamingApplication.APEX_PREFIX + "ssl.keystore.config";
```

Comment:
IMO, it will be very confusing that some settings use "dt" prefix and others use "apex" prefix. I would suggest that both "dt" and "apex" are acceptable with "dt" being a deprecated prefix similar to what was done to attributes. In the next major release "dt" can be removed. If this is acceptable to everyone, this PR should use "dt" prefix and deprecation of "dt" should be handled in a separate PR.

I also do not understand why the setting is necessary and how it is different from STRAM_HTTP_CUSTOM_CONFIG attribute. Should not it be possible to specify different keystore for different applications? How will it be covered if CUSTOM_SSL_KEYSTORE_CONFIG is not an attribute?